certification/internal/shell/scorecard_check.go

```diff
@@ -0,0 +1,66 @@
+package shell
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+	log "github.com/sirupsen/logrus"
+)
+
+type scorecardCheck struct{}
```

Comment:
Is this aiming to serve as a generic scorecard abstraction that we can then use across multiple scorecard-based checks?

Reply:
That's the idea. Otherwise, all of the code there would have been duplicated to both checks.